src/parser/MutateSentences.h

```diff
@@ -329,75 +351,142 @@ class UpdateList final {
 
 class UpdateVertexSentence final : public Sentence {
 public:
+    UpdateVertexSentence() {
+        kind_ = Kind::kUpdateVertex;
+    }
+
     void setInsertable(bool insertable) {
         insertable_ = insertable;
     }
 
-    void setVid(Expression *vid) {
-        vid_.reset(vid);
+    const bool getInsertable() const {
```

Comment:
We don't need "const" specifier for "bool"

Reply:
done!